ublox/src/ubx_packets/packets.rs

```diff
@@ -1986,6 +1986,50 @@ struct EsfMeas {
 //     msss: u32,
 // }
 
+#[ubx_packet_recv]
+#[ubx(class = 0x28, id = 0x00, fixed_payload_len = 72)]
+struct HnrPvt {
+    i_tow: u32,
+    year: u16,
+    month: u8,
+    day: u8,
+    hour: u8,
+    min: u8,
+    sec: u8,
+    valid: u8,
+    nano: i32,
+    gps_fix: u8,
```

Comment:
This one isn't marked as a bit field in the spec but it seems like it kind of is? I'ts a u8 that has 6 valid values and 2 reserved bits. Not sure if the ubx macro has something for this

Reply:
It doesn't look like a bit field, just an enum type with values 0 to 6 so not 2^8 ? can map these constants into enums with `map_type` or something

Reply:
NVM it's already implemented enum GpsFix

Reply:
Ah nice